### PR TITLE
Feature/issue 63 improve active combatant treatment

### PR DESCRIPTION
**Pull request by benjamincochran**
* Make sure to include font-weight 600 in the request to the Google Fonts API, and then explicitly use it ("bold", when 700 is included, is just too much)
* explicitly set the size of each of the elements in a combatant row so font-weight changes don't affect spacing
* Highlight the entire "active combatant" row 

Examples of the new treatment:
screen shot 2017-04-23 at 12 08 12 pm
screen shot 2017-04-23 at 12 08 23 pm

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+.sass-cache
+*.sublime-*
+local server.bat
+todo.txt
+build.js
+
+node_modules
+npm-debug.log*
+temp
+/.npmrc
```
Comment (benjamincochran) [inline .gitignore]: I have a global `.npmrc` config set for most of my projects; clobbered explicitly for KFC, but it's a good thing to ignore in general.

**Comment by Asmor** [comment]
Sorry, this slipped under my radar.

There's something funky with your changeset. Several of the files have windows-style line endings (and please change them to use unix-style line endings), but the weirdest thing is that `.gitignore` shows a bunch of lines changed which aren't, and there's no line ending difference either. Not sure what's going on with that.

image

**Comment by benjamincochran** [comment]
I'll take a look at that.

**Comment by benjamincochran** [comment]
Just now actually getting real time to look at this.  What platform have you been developing on, and what's your `core.autocrlf` setting set to?  If I compare my branch to master, I see Windows line-endings for .npmrc *without* my commits. Tested this on both Mac and Windows boxes. I've set my autocrlf settings [as recommended by Github](https://help.github.com/articles/dealing-with-line-endings/).

`git diff feature/issue-63_improve-active-combatant-treatment remotes/asmor/master`:
screen shot 2017-08-04 at 11 06 19 am

Interestingly, when I run dos2unix on these files, I get another 100% diff (with the same content). Is it possible that the the current line endings in your master are actually Windows CRLFs?

**Comment by benjamincochran** [comment]
As a follow-up: I added a .gitattributes file on my master, followed the instructions in the ["Refreshing A Repository..."](https://help.github.com/articles/dealing-with-line-endings/#refreshing-a-repository-after-changing-line-endings) guide, and it looks like your master is definitely full of CRLFs:

`git diff --ignore-space-at-eol remotes/asmor/master master` returns a ton of "warning: CRLF will be replaced by LF in XXXXXX" messages, but very few diffs; `git diff remotes/asmor/master master` returns a ton of full-file diffs.

Not sure how you want to go forward from here.

**Comment by Asmor** [comment]
I'll be honest, man. I don't know much about git, and I don't understand most of what you said. I don't know what core.autocrlf is, and if I have that file somewhere it's been generated by something and never touched.

I just checked out master and looked at a few random files. I can confirm that some of the files had unix-style endings, and some had Windows-style.

This project has had multiple contributors, and I've never run into these issues before. I'm guessing what's happened is that different files have different endings based on the peculiarities of how they were created (e.g. maybe the things created by commandline have Windows-style, while the ones I've created in Sublime have unix style which is my default setting). Presumably most other people are like me, in that they have their editor configured to automatically detect the endings that the file uses and just use that, even if it's inconsistent between different files. It seems like you have some things set to enforce consistent line endings everywhere, and that may be what's causing the issue.

Anyways, the bottom line is that making the line endings in the project consistent is out of scope of this issue, and while it's certainly not good that they're inconsistent fixing that isn't a big priority for me. I'd be willing to consider a pull request for addressing that issue specifically, if you care enough to tackle it.

As far as this issue goes, I'm going to have to request that you clean up the diffs and maintain the consistency of line endings in each file as they currently exist before I can merge it. Sorry.

**Comment by jbarzilai** [comment]
Yeah, sorry I haven't chimed in on this one. I'm pretty sure the Windows
line endings may be my fault. I typically have my git settings on "Don't
mess with it, in or out" so I may have inadvertently added a mess load of
Windows line endings during the massive refactoring.

Sorry if I did and caused a headache.

**Comment by Asmor** [comment]
I don't think it's you. Or, at least, it's not _just_ you. The `.gitignore` has windows endings, and I definitely made that file and would have noticed if a pull request had altered the endings of it, like this one has.

**Comment by rgant** [comment]
Not really part of this issue as Asmor notes, but if you do want to force unix line endings all the time (which I really recommend) then I found this response useful to setting that up and fixing a repo: https://stackoverflow.com/a/9977954

**Comment by benjamincochran** [comment]
Agreed that fixing the line-ending stuff is outside the scope of #63.  I'll submit a separate PR for that.

_--edit--_ https://github.com/Asmor/5e-monsters/pull/74

**Comment by benjamincochran** [comment]
I think this should be better now.

**Comment by Asmor** [comment]
Thanks, it's merged and live!

**Comment by benjamincochran** [comment]
Thanks guys. Love this project;  sorry this was such a drawn-out hassle!

**Comment by benjamincochran** [comment]
Hmm, seeing it merged, yes, but not yet live :-/

**Comment by Asmor** [comment]
Woops. I have a copy of it that I use for testing, and I only updated that one. Should be live now. Sorry about that.

**Pull request merged**